lib/webhooks/__tests__/register.test.ts

```diff
@@ -357,6 +366,34 @@ describe('shopify.webhooks.register', () => {
     );
     assertRegisterResponse({registerReturn, topic, responses});
   });
+
+  it('returns which operation was done for each handler', async () => {
+    shopify.webhooks.addHandlers({
+      PRODUCTS_UPDATE: {...HTTP_HANDLER, includeFields: ['id', 'title']},
+    });
+    shopify.webhooks.addHandlers({
+      PRODUCTS_DELETE: HTTP_HANDLER,
+    });
+
+    queueMockResponses(
+      [JSON.stringify(mockResponses.webhookCheckMultiHandlerResponse)],
+      [JSON.stringify(mockResponses.successResponse)],
+      [JSON.stringify(mockResponses.successUpdateResponse)],
+      [JSON.stringify(mockResponses.successDeleteResponse)],
+    );
+
+    const registerReturn = await shopify.webhooks.register({session});
+
+    expect(registerReturn.PRODUCTS_CREATE[0].operation).toEqual(
+      WebhookOperation.Delete,
+    );
```

Comment:
Yeah this is a little confusing, but in our test response, we're returning a pre-existing `PRODUCTS_CREATE` webhook and a `PRODUCTS_UPDATE` one, so we create the delete webhook, and delete the create one 😅

Reply:
what might be throwing me off is why is `PRODUCTS_CREATE` capitalized? it looks like a dynamic list so it's strange to see it as a constant. looking at the second assertion, the `PRODUCTS_DELETE` returns an operation that's a `Create`.

Reply:
So the capitalized versions are the actual topics we're registering, which match the values we allow in the API - so `PRODUCTS_CREATE` is the topic we're registering, whereas `Create` is the operation.

One thing I could do would be to use a topic other than `PRODUCTS_DELETE`, maybe something like `ORDERS_CREATE` as a topic. That might reduce the confusion.

Reply:
as discussed in our pairing, `WebhookOperation` should have `subscribe` and `unsubscribe` instead of `Create` and `Delete` so that we don't possibly confuse the creation of the webhook with the action related to the topic

Reply:
I've added a TODO with a note for us to rename the enum values on the next major version. This type is extremely unlikely to be used by any apps, so it shouldn't really break anything, but it's better to be safe since it will be a change to the package's public API.